supported_formats_and_operations.md

```diff
@@ -23,95 +23,96 @@ Which consumer software supports which format?
     - `std` the standard deviation values to use for `mode == fixed`. Analogous to `mean`.
-    - `[eps]` epsilon for numeric stability: `out = (tensor - mean) / (std + eps)`.
+    - `[eps]` epsilon for numeric stability: `out = (tensor - mean) / (std + eps)`. Default value: 10e-7.
   - `reference_implementation`
-
+  
 ## Consumers
 
-Which consumer supports which preprocessing operation?
+Which consumer supports which pre-/postprocessing operation?
 
-| preprocesing               | ilastik | deepImageJ | Fiji |
+| pre-/postprocesing         | ilastik | deepImageJ | Fiji |
 | -------------------------- | ------- | ---------- | ---- |
+|  `binarize`                | no      | ?          | ?    |
 |  `clip`                    | ?       | ?          | ?    | 
-|  `min_max`                 | ?       | ?          | ?    |
-|  `percentile`              | ?       | ?          | ?    |
-| `scale_linear`             | ?       | ?          | ?    |
+|  `scale_linear`            | ?       | ?          | ?    |
+|  `sigmoid`                 | yes     | ?          | ?    |
 |  `zero_mean_unit_variance` | yes     | ?          | ?    |
 
 
-# Postprocessing
+## Preprocessing
 
-The supported postprocessing operations.
+Additional preprocessing operations.
 
-- `binarize` binarize the tensor with a fixed threshold, values above the threshold will be set to one, values below the threshold to zero
+- `scale_range` normalize the tensor with percentile normalization
   - `kwargs`
-    - `threshold` the fixed threshold
-  - `reference_implemation`
-- `clip` clip the tensor, same as in [preprocessing](https://github.com/bioimage-io/configuration/blob/master/supported_formats_and_operations.md#preprocessing).
-- `sigmoid` apply a sigmoid to the tensor.
-  - `kwargs` None
-  - `reference_implementation`
-- `scale_mean_variance` scale the tensor s.t. its mean and variance match a reference input 
+    - `mode` can be one of `per_sample` (percentiles are computed for each sample individually), `per_dataset` (percentiles are computed for the entire dataset). For a fixed scaling use `scale linear`.
+    - `axes` the subset of axes to normalize jointly. For example `xy` to normalize the two image axes for 2d data jointly. The batch axis (`b`) is not valid here.
+    - `min_percentile` the lower percentile used for normalization, in range 0 to 100. Default value: 0.
+    - `max_percentile` the upper percentile used for normalization, in range 0 to 100. Has to be bigger than `upper_percentile`. Default value: 100.
+  - `reference_implementaion`
+
+
+### Consumers
+
+Which consumer supports which preprocessing operation?
+
+| preprocesing               | ilastik | deepImageJ | Fiji |
+| -------------------------- | ------- | ---------- | ---- |
+|  `scale_range`             | ?       | ?          | ?    |
+
+
+## Postprocessing
+
+Additional postprocessing operations.
+
+- `scale_range` normalize the tensor with percentile normalization
```

Comment:
We should add something like "Same keyword arguments as preprocessing:scale_range + reference_tensor"

Reply:
there is ... 

```
`kwargs`: 
  - same as preprocessing
  - `reference_tensor`
``` 
or am I misunderstanding you here?

Reply:
ah sorry, I just overlooked that.